src/app/components/source/UserSecurity.js

```diff
@@ -492,17 +497,17 @@ class UserSecurity extends Component {
         <h2 style={style}>
           <FormattedMessage id="userSecurity.changePassword" defaultMessage="Change password" />
         </h2>
-        <StyledPasswordChange>
-          <Card style={style} className="user-password-change__card">
+        <ContentColumn center className="user-password-reset__component">
+          <StyledCard>
```

Comment:
I guess a regular `Card` will do here, no need for the extra top padding added by `StyledCard`.

Reply:
Replaced `StyledCard` with `Card`